citrination_client/client.py

```diff
@@ -304,6 +304,35 @@ def upload_file(self, file_path, dataset_id, root_path=None):
         """
         return self.upload(dataset_id, file_path)
 
+    def _data_analysis(self, model_name):
```

Comment:
I think nearly this exact method exists in `feature/dev-api` as a public method (and is used in the api tests). Should we keep this private or let users consume data analyses directly?